src/utils.ts

```diff
@@ -53,4 +53,29 @@ function getCheckText(trivyStatus: number, dockleStatus: number): string {
     }
 
     return text;
+}
+
+export function getConfigForTable(col1_width: number, col2_width: number, col3_width: number, col4_width: number): any {
+    let config = {
```

Comment:
Do you want to keep this dynamic? (Pass an array columnWidths and return the config accordingly. That way addition/deletion of columns would be simple in future.

Reply:
Replaced with an array